### DISCUSSION TITLE
Automated checkbox field for gifted preparations

### DISCUSSION THREAD

**Discussion by bronwyncombs** [specify/specify7#5965] [Feature Requests]
**Is your feature request related to a problem? Describe the solution you'd like**

> First, my complements and thanks for the new “Deaccession” function.  I was able to successfully create a new Deaccession record from within an existing Gift record.
>  
> I now request an enhancement to the Preparations section of the Collection Object screen to highlight when a Preparation item is ‘gifted’ (permanently given to another entity).  
>
> I suggest that the current processing of a “loan” and the use of the “On Loan” check box be duplicated with a “Gift” checkbox for each Preparation item “gifted” that would be automatically  ‘checked’ when a new GIFT record is created and saved.
> 
> <img width="762" alt="Screenshot 2024-07-08 at 3 15 08 PM" src="https://github.com/specify/specify7/assets/135047322/dd268895-c417-484c-9b1d-adf03c856d21">

---

Example using yesNo2 and yesNo3 from the preparation table: 

<img width="1190" alt="Screenshot 2024-07-08 at 2 34 13 PM" src="https://github.com/specify/specify7/assets/135047322/beab020e-9aa6-4baa-b435-08e61e7dd721">

Note: Gifts are already shown in the "Show Loans" (Show Interactions) dialog: 

<img width="1190" alt="Screenshot 2024-07-08 at 2 34 22 PM" src="https://github.com/specify/specify7/assets/135047322/fc07052b-36d0-4156-bf42-99b8825d439b">

---
**Reported By**
Randy Williams @ OSU

**Comment by specifysoftware** [comment]
This issue has been mentioned on **Specify Community Forum**. There might be relevant details there:

https://discourse.specifysoftware.org/t/add-a-gifted-checkbox-to-the-preparations-screen-request-5082/1864/2

**Comment by bhumikaguptaa** [comment]
> Hi support!
A user of JACA Herbaria (specify.ipe.csic.es) is requesting a checkbox similar to the IsOnLoan on the preparations form but for Gifts.  I see IsOnLoan checbox is based on a virtual field so the request would be the virtual field itself (IsOnGift).

Reported by: Silvia Lusa, CSIC, https://specifysoftware.bolddesk.com/agent/tickets/559

**Comment by g1rly-c0d3r** [comment]
Linked in https://github.com/specify/specify7/issues/8174
